apps/angular/projection/src/app/component/teacher-card/teacher-card.component.ts

```diff
@@ -1,35 +1,38 @@
 import { Component, OnInit } from '@angular/core';
 import { FakeHttpService } from '../../data-access/fake-http.service';
+import { Store } from '../../data-access/store';
 import { TeacherStore } from '../../data-access/teacher.store';
 import { CardType } from '../../model/card.model';
 import { Teacher } from '../../model/teacher.model';
 import { CardComponent } from '../../ui/card/card.component';
 
 @Component({
   selector: 'app-teacher-card',
-  template: `<app-card
-    [list]="teachers"
-    [type]="cardType"
-    customClass="bg-light-red"></app-card>`,
-  styles: [
-    `
-      ::ng-deep .bg-light-red {
-        background-color: rgba(250, 0, 0, 0.1);
-      }
-    `,
-  ],
+  template: `
+    <app-card
+      [list]="teachers"
+      [type]="cardType"
+      customClass="bg-red-200/50"
+      showingProp="firstName">
+      <img src="assets/img/teacher.png" alt="" />
+    </app-card>
+  `,
   standalone: true,
   imports: [CardComponent],
+  providers: [{ provide: Store, useExisting: TeacherStore }],
```

Comment:
Ok I got why you did that. That's an idea, but too complex. Let the logic inside this component, and the card component will only dispatch add and delete action